linting/extra/src/strict_balance_equality.rs

```diff
@@ -224,8 +224,6 @@ impl<'a, 'tcx> AnalysisDomain<'tcx> for StrictBalanceEqualityAnalysis<'a, 'tcx>
 
     const NAME: &'static str = "strict_balance_equality";
 
-    type Direction = Forward;
```

Comment:
The default `Direction` is `Forward`, see: https://github.com/rust-lang/rust/issues/120130